textanalytics/low-level-dynamic/azure-ai-textanalytics-models/src/samples/java/com/azure/ai/textanalytics/RecognizeLinkedEntitiesWithModels.java

```diff
@@ -33,9 +33,9 @@ public static void main(String[] args) {
                 .setId("0").setText("Old Faithful is a geyser at Yellowstone Park.");
         MultiLanguageBatchInput batchInput = new MultiLanguageBatchInput().setDocuments(Collections.singletonList(input));
 
-        EntityLinkingResult result = client.entitiesLinking() // DynamicRequest
+        EntityLinkingResult result = client.getLinkedEntities() // DynamicRequest
                 .setBody(batchInput) // DynamicRequest
-                .context(Context.NONE) // DynamicRequest
+                .setContext(Context.NONE) // DynamicRequest
```

Comment:
Don't we plan to remove the setContext method?

Reply:
Yeah, based on offline discussion, we decided to have `send()`, `send(Context context)` and `sendAsyc()` (async uses Reactor context).